jenkins-design-language/pom.xml

```diff
@@ -15,6 +15,14 @@
 
     <url>https://wiki.jenkins-ci.org/display/JENKINS/Jenkins+Design+Language</url>
 
+    <dependencies>
+        <dependency>
+           <groupId>io.jenkins.blueocean</groupId>
+           <artifactId>blueocean-commons</artifactId>
```

Comment:
Why is this added? JDL should not depend on this.

Reply:
@kzantow 

> if jenkins-design-language does not have any Maven dependency with core plugins of Blue Ocean, this plugin could be installed independently of the rest of Blue Ocean plugins. This would be an not desired behavior.

If this plugin is completely independent and has its own lifecycle, please move it out side of this multi module project and use a version naming not synced with Blue Ocean.

Reply:
@recena Do not get this change. Yes this plugin has no dependency on other modules but its tied to blueocean release lifecycle and its intentional. 

>please move it out side of this multi module project and use a version naming not synced with Blue Ocean.

What motivates this change request?

Reply:
@recena not sure if depending on commons will solve anything - it will just get people to install more unrelated plugins than just jdl